libraries/chain/transaction.cpp

```diff
@@ -343,10 +344,35 @@ signed_transaction packed_transaction::get_signed_transaction() const
 
 }
 
-void packed_transaction::set_transaction(const transaction& t, packed_transaction::compression_type _compression)
+packed_transaction::packed_transaction( bytes&& packed_txn, vector<signature_type>&& sigs,
```

Comment:
I think it would be clearer if this constructor was split into two:

- `packed_transaction( bytes&& packed_txn, vector<signature_type>&& sigs,  bytes&& packed_cfd, compression_type _compression )`
- `packed_transaction( bytes&& packed_txn, vector<signature_type>&& sigs,  vector<bytes>&& cfd, compression_type _compression )`

Reply:
Fixed in #6494